src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -46,8 +46,8 @@
 // whether they can talk to a given wallet without having to know in
 // advance which version they will stop working with
 // Don't go over 32767 for any of these
-#define WALLET_RPC_VERSION_MAJOR 2
-#define WALLET_RPC_VERSION_MINOR 0
+#define WALLET_RPC_VERSION_MAJOR 1
+#define WALLET_RPC_VERSION_MINOR 24
```

Comment:
Bumps minor version from 23 -> 24.

Reply:
Note: need to revisit this when merging the PR, in case I rebase onto a higher version number.